repose-aggregator/tests/performance-tests/roles/gatling/files/gatling.conf

```diff
@@ -0,0 +1,132 @@
+#########################
+# Gatling Configuration #
```

Comment:
I've been torn about whether or not we should include the whole config file. I like having all of the possible config options right here, but it makes it difficult to see what we've configured (i.e. changed from the default value).

If that's just me and you think it's a great idea to leave everything here, then that's fine with me.